docs/templates/40-updating.ftl

```diff
@@ -0,0 +1,3 @@
+# Updating
```

Comment:
This section can either be removed or we can simply provide a link to https://jenkins.io/doc/book/managing/plugins/